selfdrive/ui/replay/replay.cc

```diff
@@ -0,0 +1,70 @@
+#include "replay.hpp"
+
+Replay::Replay(QString route_, int seek) : route(route_) {
+  unlogger = new Unlogger(&events, &events_lock, &frs, seek);
+  seg_add = 0;
+
+  http = new HttpRequest(this, "https://api.commadotai.com/v1/route/" + route.replace("/", "|") + "/files", "Route_HttpRequest");
```

Comment:
it was entered with the pipe, and now you're just reverting something done in replay/main.cc